tests/integration/review_app/test_app_df_graph.py

```diff
@@ -7,9 +7,15 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import dash_table
+import time
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
```

Comment:
Now you can get rid of these too, right? Which I'd have thought flake8 would have told you but perhaps it's not run on the test code...

Reply:
You're right, it's apparently not run on the tests folder

Reply:
https://github.com/plotly/dash-table/pull/713/commits/a77b1592c1caef144be2a64f380a05e3e5291c8e